.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,47 @@
+name: Bug Report
+description: File a bug/issue
+title: 'bug: <title>'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! The more info you provide, the more we can help you 🙌
+
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
```

Comment:
`I have looked through the [existing issues](https://github.com/scaffold-eth/scaffold-eth-2/issues)`

Reply:
https://github.com/scaffold-eth/scaffold-eth-2/pull/329/commits/b428328d56ed8d57e360374a95ecc16138eca7cf